### ISSUE TITLE
Remove old Go SDK

### ISSUE THREAD

**Issue by danielrbradley** [pulumi/pulumi-azure-native#2192]
The Go SDK at `~/sdk/go` is no longer generated or published because it's too large to download via Go's package cache. The's a new SDK now available from https://github.com/pulumi/pulumi-azure-native-sdk

The old SDK is still remaining in the main repository for reference for now, but we should remove this and reduce the size of checkouts.

**Comment by sanmai-NL** [comment]
@danielrbradley What's the reasoning behind the ‘for reference’? One can use Git history for this, can't one?

**Comment by danielrbradley** [comment]
Yes it'll still be there in the history, but I think some users tend to just browse the worktree via GitHub directly. Hopefully just leaving a helpful link will be enough to guide users in the right direction.
